.travis.yml

```diff
@@ -26,9 +32,12 @@ before_install:
   - npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
   # Use latest supported NPM in order to speedup build and support `npm ci`
   - npm i -g npm@6
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
 
 install:
   - npm ci
+  - npm install
```

Comment:
Why do we need `npm install` after `npm ci` which is installing dependencies already?